oci/table_oci_ons_subscription.go

```diff
@@ -275,25 +280,15 @@ func getSubscriptionDeliveryPolicy(ctx context.Context, d *plugin.QueryData, h *
 		return nil, err
 	}
 
-	get := h.Item.(ons.Subscription)
+	subscriptionItem := h.Item.(ons.Subscription)
 	request := ons.ListSubscriptionsRequest{
 		CompartmentId: types.String(compartment),
-		TopicId:       types.String(*get.TopicId),
-		Limit:         types.Int(50),
```

Comment:
What is the default `Limit`? I think it's ok to leave a Limit here, my review comment was primarily around not respecting the SQL query `limit`